src/test/scala/chiselTests/BundleSpec.scala

```diff
@@ -8,32 +8,28 @@ import chisel3.testers.BasicTester
 
 trait BundleSpecUtils {
   class BundleFooBar extends Bundle {
-    val foo = UInt(16.W)
-    val bar = UInt(16.W)
+    val foo                           = UInt(16.W)
+    val bar                           = UInt(16.W)
     override def cloneType: this.type = (new BundleFooBar).asInstanceOf[this.type]
```

Comment:
```suggestion
    val foo = UInt(16.W)
    val bar = UInt(16.W)
```

Reply:
oh noes i turned on my code formatter will clean this up.

That was there and I nicely got a compiler error of `msg` :)